.github/workflows/integration_tests.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: [ '3.7', '3.8', '3.9', '3.10' ]
```

Comment:
3.7 is no longer being supported and support for 3.8 is ending this year: https://devguide.python.org/versions/

Reply:
Can you no longer install on python 3.7 /8 now? Or is it just not being tested.

Reply:
Just out of an abundance of caution we might want to check what version of python the terra and all of us images are using.

Reply:
Terra/AOU uses python3.10.12
cromshell on p3.7 is no longer being tested

I had some trouble installing dev packages on py3.11 because  mypy==0.770 was not supported. 
So I updated this to the latest [version](https://pypi.org/project/mypy/), the latest version doesn't work on 3.7. None of the packages in the requirements have been updated so cromshell should still work on py3.7 but if anyone is developing on 3.7 they'll run into package issues.